src/crawl4ai_mcp.py

```
@@ -69,18 +69,20 @@ def main():
     parser = argparse.ArgumentParser(description="Crawl4AI MCP Server")
     parser.add_argument('--host', default=os.getenv("HOST", "0.0.0.0"), help='Host to bind to')
     parser.add_argument('--port', type=int, default=int(os.getenv("PORT", 8051)), help='Port to bind to')
+    parser.add_argument('--transport', default=os.getenv("TRANSPORT", "http"), help='Transport to use (http, sse, stdio)')
+    parser.add_argument('--path', default=os.getenv("MCP_PATH", "/mcp/"), help='Path for HTTP transport')
     
     args = parser.parse_args()
-    logger.info(f"🔧 Server args: host={args.host}, port={args.port}")
+    logger.info("🔧 Server args: host=%s, port=%s, transport=%s, path=%s", args.host, args.port, args.transport, args.path)
     
     # Setup signal handlers for graceful shutdown
     def signal_handler(signum, frame):
```

Ruff (0.12.2)

Missing return type annotation for private function `signal_handler`

Add return type annotation: `None`

(ANN202)